### PR TITLE
*: Add some comments about decoding

### PR DESCRIPTION
**Pull request by JaySon-Huang**
Signed-off-by: JaySon-Huang <jayson.hjs@gmail.com>

### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflash/issues/4862

### What is changed and how it works?

 * Add some comments about decoding row data

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by breezewish — APPROVED** [review]
Approving as there is no logic changes.

File: tests/fullstack-test/mpp/mpp_fail.test [hunk 1]
```diff
@@ -21,6 +21,7 @@ mysql> insert into test.t values(1,'a'),(2,'b'),(3,'c')
 mysql> alter table test.t set tiflash replica 1
 
 func> wait_table test t
+mysql> analyze table test.t
```
Comment (JaySon-Huang) [inline tests/fullstack-test/mpp/mpp_fail.test:24]: Add this line to fix the error caused by tidb execute plan change
```
[2022-06-17T17:32:18.320Z]   File: fullstack-test/mpp/mpp_fail.test
[2022-06-17T17:32:18.320Z]   Error line: 30
[2022-06-17T17:32:18.320Z]   Error: use test; set @@tidb_isolation_read_engines='tiflash'; set @@tidb_allow_mpp=1; select count(value), id from t group by id;
[2022-06-17T17:32:18.320Z]   Result:
[2022-06-17T17:32:18.320Z]     count(value)	id
[2022-06-17T17:32:18.320Z]     1	2
[2022-06-17T17:32:18.320Z]     1	3
[2022-06-17T17:32:18.320Z]     1	1
[2022-06-17T17:32:18.320Z]   Expected:
[2022-06-17T17:32:18.320Z]     ERROR 1105 (HY000) at line 1: DB::Exception: Fail point FailPoints::exception_before_mpp_register_non_root_mpp_task is triggered.
```

**Pull request merged**